### DISCUSSION TITLE
Can I "@apply" DaisyUI?

### DISCUSSION THREAD

**Discussion by fballiano** [saadeghi/daisyui#743] [Q&A] [answered]
Hi everybody,
using DaisyUI as a tailwind plugin, is it possible to do something like this:

```
button {
    @apply btn btn-primary;
}
```

it doesn't work but I'm not sure if I'm miss something in the setup of the whole project.

I know it's maybe a weird way or working with DaisyUI but in my usecase I cannot modify the HTML templates adding the classes.

Thanks a lot!

**Comment by saadeghi** [comment, marked as answer]
It should work:  
https://play.tailwindcss.com/sYjQewggoU?file=css

I also tested on a Vite project

**Reply by fballiano** [reply]
I'm so sorry, rebuilding the tailwind.config it worked perfectly, I don't know what I did wrong the first time.

Thanks so much!
